Tests/CMakeLists.txt

```diff
@@ -96,6 +96,14 @@ function(add_test_r TEST_NAME TEST_EXE_DIR)
     set_tests_properties(${TEST_NAME} PROPERTIES TIMEOUT 18000 PROCESSORS ${PELE_NP} WORKING_DIRECTORY "${CURRENT_TEST_BINARY_DIR}/" LABELS "regression" ATTACHED_FILES "${CURRENT_TEST_BINARY_DIR}/${TEST_NAME}.log")
 endfunction(add_test_r)
 
+#Standard regression test with looser tolerances
+function(add_test_rtol TEST_NAME TEST_EXE_DIR)
+    setup_test()
+    set(RUNTIME_OPTIONS "amr.max_step=10 mac_proj.atol=5e-11 mac_proj.rtol=5e-11 mac_proj.verbose=2 ${RUNTIME_OPTIONS}")
```

Comment:
Is this still necessary? 

I think the better strategy might be to loosen the tolerances directly in the input file for the affected cases rather than create this additional test type.